### PR TITLE
Add ability to export to Normandy - Fixes #1515

### LINKED ISSUE #1515: Link the "Send To Normandy" button to the new Delivery Console import page
Delivery Console will support experiment imports very soon and we should link the button to DC (`{DC_ROOT_URL}/recipe/import/{experiment_slug}`).

### PR DESCRIPTION
**Pull request by rehandalal**
r?

New modal for "Send to Normandy" button:
image

After clicking the "Already exported" button:
Screenshot from 2019-08-08 02-39-04

Preview link in Normandy Info:
image

### CODE REVIEW

**Comment by rehandalal** [comment]
@uhlissuh tagging you in as well since you worked on some of the code i tweaked

**Comment by uhlissuh** [comment]
This is cool, I'm down with these changes... all looked good.

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ test_build: build
 	docker-compose -f docker-compose-test.yml build
 
 test: test_build
-	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- pytest --cov --cov-report term-missing"
+	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- pytest -vvvv --cov --cov-report term-missing"
```
Comment (jaredkerim) [inline Makefile]: Was this for your local testing? Should we be committing this for everybody?

Comment (rehandalal) [reply Makefile]: it was for my local testing but probably good for everyone. maybe it should be in a different commit/pr

**Pull request merged**